src/dot_dotfiles/zsh/snippets/oh-my-zsh.zsh.tmpl

```diff
@@ -50,10 +38,15 @@ if [[ -n "$TMUX" ]]; then
     DISABLE_AUTO_TITLE="true"       # Disable auto-title in tmux for performance
 fi
 
-# =============================================================================
-# LOAD OH-MY-ZSH
-# =============================================================================
-source $ZSH/oh-my-zsh.sh
+# Set the ZSH environment variable to the chezmoi-managed location
+export ZSH="$HOME/.oh-my-zsh"
```

Comment:
since we manage this, im thinking we install this to a different directory that doesnt pollute the home dir. come up with a better location we can deploy to